libraries/azure-app-configuration-importer/src/internal/parsers/defaultConfigurationSettingsConverter.ts

```diff
@@ -193,14 +271,15 @@ class FeatureFlagConfigurationSettingsConverter implements ConfigurationSettings
     }
   }
 
-  private getFeatureFlagValue(
+  private getFeatureFlagValueFromDotnetSchema(
     featureFlagName: any,
```

Comment:
nit: featureFlagName isn't a string ?

Reply:
I am thinking it can only be a string, but this is what we had previously in the code. I am not sure if there was a reason why we decided to have it as any

Reply:
I don't think there was a reason to have it as any, we can update type to have it as string

Reply:
Updated this type